react-charts/bullet-chart/step1.md

```diff
@@ -1,10 +1,18 @@
-PatternFly React charts are based on the [Victory chart](https://formidable.com/open-source/victory/docs/victory-chart/) library, along with additional functionality, custom components, and theming for PatternFly. 
-This provides a collection of React based components you can use to build PatternFly patterns with consistent markup, styling, and behavior. 
-
-PatternFly React charts are composed using separate components, so features are more easily customizable. 
-In this course, we're going to build a PatternFly bullet chart together - starting with a simple chart, adding qualitative ranges, primary comparative measures, a comparative warning measure, tooltips, labels, a legend, and concluding by changing the theme color.
-You'll learn how to use PatternFly React chart components together to build a consistent user experience.
-
-The image below is what we'll end up with when we are done.
-
-<img src="bullet-chart/assets/final.png" alt="Completed chart component" style="box-shadow: rgba(3, 3, 3, 0.2) 0px 1.25px 2.5px 0px;" />
+Begin by adding a CSS selector to define the chart size.
+
+Katacoda is setting up a new React application. Begin coding once the server starts and "Welcome to PatternFly" is visible on the lower pane.
+
+1. <strong>Navigate to the `src` folder and open `src/app.css`{{open}}<strong>
```

Comment:
Change all the numbers to 1) -- in katacoda the numbers get rendered like markdown and have a large space to the left which leaves less room